app/models/searchgov_domain.rb

```diff
@@ -71,6 +75,13 @@ def sitemap_urls
     urls.presence || ["#{url}sitemap.xml"]
   end
 
+  def stop_indexing!
+    done_indexing
+    self.status = INDEXING_STOPPED
```

Comment:
do we need to explicitly declare self here?

Reply:
Yes!, we do. If not it only creates a `status` variable for the method